few/lib/epsilon_lexicase.h

```diff
@@ -96,9 +96,8 @@ void epsilon_lexicase(const ExtMat & F, int n, int d,
   for (int i = 0; i<epsilon.size(); ++i)
     epsilon(i) = mad(F.col(i));
 
-vector<int> ind_locs(n);
-if(lex_size){
-    char message[200];
+  vector<int> ind_locs(n);
```

Comment:
yes, I also thought so, but it was showing an error with just declaring it above vector<int> ind_locs and then reserving size inside else statement(when lex_size is not there) created an issue. And if I just declare and define inside else, it may not be accessible outside.
In case this is 2n after lex_size and is an issue, will have to see an alternative.